config/initializers/download.rb

```diff
@@ -0,0 +1,39 @@
+module Geoblacklight
+  class Download
+  ##
+  # Monkey patch of Geoblacklight::Download that injects geoserver auth
+  # credentials for restriced princeton wms layers. Delete this as soon as 
+  # geoblacklight downloads become configurable or multiple direct downloads
+  # are integrated into the document schema.
```

Comment:
Should I assume that the blocking upstream issue here is https://github.com/geoblacklight/geoblacklight/issues/548?

Reply:
Yep. Those are the ones. I'm thinking that we could just generate downloadable KML, GeoJSON derivatives instead of offloading that to GeoServer.